ZTors.tex

```diff
@@ -1102,6 +1102,78 @@ \subsection{Induction in $\TorZ$}\label{sec:TorZ-induction}
+Her main theorem~\cite[Theorem 50]{sojakova:hits-hias} then implies that if $C$ has $\dS$-induction, then it is \emph{homotopy-initial} in that for any other $\dS$-algebra $(A,a,l)$ the type of ``$\dS$-homomorphisms''
+\[H_{C,A} \defeq \sum_{h:C\to A} \sum_{\beta: h(c) = a} \ap{h}(s) \ct \beta = \beta \ct l  \]
+is contractible, i.e.\ there is an element of the type
+\[ \prod_{(A,a,l)} \mathrm{Contr}(H_{C,A}). \]
+It follows that our $\TorZ$ is homotopy-initial in this sense.
+
+Now suppose we have a Quillen model category $\cE$ that presents an $(\infty,1)$-topos $\sE$.
+To show that $\TorZ$ has the universal property of a circle object in $\sE$, we must show that for any object $A$ of $\sE$, the diagram of hom-spaces ($\infty$-groupoids)
+\[ \sE(\TorZ,A) \to \sE(1,A) \rightrightarrows \sE(1,A) \]
+is a homotopy equalizer, i.e.\ that the map from $\sE(\TorZ,A)$ to the homotopy equalizer of two copies of the identity map of $\sE(1,A)$ is an equivalence.
+It suffices to show that the homotopy fiber of this map over any point is contractible, which is to say that given any point $a:1\to A$ and homotopy $l:a\sim a$, the space of maps $h:\TorZ\to A$ equipped with a homotopy $\beta : h(\pt) \sim a$ and a higher homotopy $h(\Zloop) \ct \beta \sim \beta \ct l$ is contractible.
+
+Now, homotopies in the $(\infty,1)$-category $\sE$ can always be presented by \emph{right homotopies} in the model category $\cE$, meaning maps into a path object.
+Thus, our object $A$ of $\sE$ with $a$ and $l$ can be presented by an object of $\cE$, which we also denote $A$, with a point $a:1\to A$ in $\cE$ and a right homotopy $l:1\to P A_{(a,a)}$, where $P A$ is the path object of $A$ and $P A_{(a,a)}$ is its pullback along $(a,a):1\to A\times A$.
+Since path objects supply the interpretation of identity types, this corresponds to a type $A$ with an element $a:A$ and path $l:a=_A a$, i.e.\ an $\dS$-algebra.
+By a similar argument, the homotopy fiber that we want to prove contractible is equivalent to the hom-space $\sE(1, H_{\TorZ,A})$.
+Thus, it suffices to observe that if $B$ is an object with a point $1\to \mathrm{Contr}(B)$ then $B$ is equivalent to the terminal object (a detailed proof can be found in~\cite[Lemma 4.1]{shulman:elreedy}).
+
+It is also worth saying a few words about why our $\TorZ$ ``classifies $\ZZ$-torsors'' in the $(\infty,1)$-categorical sense, i.e.\ that for any object $A$, the hom-space $\sE(A,\TorZ)$ is naturally equivalent to the $\infty$-groupoid of small $\ZZ$-torsors in $\sE/A$.
+Importantly, the relevant notion of ``$\ZZ$-torsor'' is the topos- and sheaf-theoretic one: a $\ZZ$-torsor $X$ in $\sE/A$ need not be ``globally'' isomorphic to $\ZZ\times A$, but only ``locally'' so in the sense that there is an effective epimorphism $p:B \twoheadrightarrow A$ such that $p^*X$ is isomorphic over $B$ to $\ZZ\times B$.
```

Comment:
"topos-" isn't a word.  What about "topos-theoretic" instead?

Reply:
It's a [suspended hyphen](https://en.wikipedia.org/wiki/Hyphen#Suspended_hyphens).

Reply:
Okay.